packages/ts-transform/src/utils/css-transform.tsx

```diff
@@ -19,33 +20,42 @@ const minify = () => {
     });
 };
 
-const COMMA_CHAR_CODE = 58;
+const removeSpacesBeforeSelector = (selector: parser.Node) => (selector.spaces.before = '');
+const doesCombinatorTypePrecedesSelector = (selector: parser.Node) => {
+  const previousSelector = selector.prev();
 
+  return previousSelector && previousSelector.type === 'combinator';
+};
+const prependNestingTypeToSelector = (selector: parser.Node) => {
+  const { parent } = selector;
+
+  if (parent) {
+    const nesting = parser.nesting({});
```

Comment:
```suggestion
    const nesting = parser.nesting();
```

I don't think we need the empty object according to api docs https://github.com/postcss/postcss-selector-parser/blob/master/API.md#parsernestingprops

Reply:
I know, but typings were throwing error. Typings seems to be messed up. Options were marked required 😞

Reply:
oh 🤔 

can we copy their latest types from that PR and just make our own typedef?

Reply:
sure